pennylane/capture/switches.py

```diff
@@ -62,3 +63,22 @@ def status_fn() -> bool:
 
 
 enable, disable, enabled = _make_switches()
+
+
+@contextmanager
+def stop_recording():
```

Comment:
I feel like it's a good idea to move away from queuing related semantics. Perhaps `pause_binding_ops`?